example-gto/code/mlem/train.py

```diff
@@ -0,0 +1,17 @@
+import random
+
+import emoji
+import mlem
+
+def model(text: str):
+    """
+    Translate dog barks to emoji, as you hear them
+    """
+    return " ".join(
+        random.choice(list(emoji.EMOJI_DATA.keys())) for _ in text.split()  # type: ignore
+    )
+
+
+if __name__ == '__main__':
+
+    mlem.api.save(model, "models/churn.pkl", sample_data="Woof woof!", external=True)
```

Comment:
In my POC this script had a parameter that actually changed model. This allows to play with repo a lot easier: create new model versions, commit them and see them deployed

Reply:
I can get it back if you think it's a useful part. Didn't consider that usecase before. In that case, we also need to add some instructions on how to use it. I think we can add them on the MLEM/GTO integration page maybe? E.g. show that we change it, train new model, then deploy and check it's now a different one. WDYT? The other option is repo's README, but that doesn't sound good to me.

Reply:
maybe both

Reply:
Ok, I returned your model back. DIdn't include model deploy/test from your `run.sh`, and `sample` data your saved.

Reply:
> MLEM - not much about in the get started so far? do we need to add something about the deployment? Do we want to include at least a link to GH action? what would be the best way for people to try it?

@shcheklein, don't think it worth adding neither to MLEM's README nor to MLEM GS. I think we need a page in MLEM's docs (nearby with DVC integration page). And we can put a link to it in the `example-gto` repo README. WDYT?

Reply:
Yep sounds good to me!